WalletLibrary/WalletLibrary/Networking/Entities/OpenID4VCI/CredentialMetadata.swift

```diff
@@ -44,6 +62,43 @@ extension CredentialMetadata
             }
         }
     }
+    
+    /// Defines a function to get credentials configurations with given ids.
+    func getCredentialConfigurations(ids: [String]) -> [CredentialConfiguration]
+    {
+        var configurations: [CredentialConfiguration] = []
+        for id in ids
+        {
+            if let configuration = credential_configurations_supported?[id]
+            {
+                configurations.append(configuration)
+            }
+        }
+        
+        return configurations
+    }
+    
+    /// Defines a function to get localized Requester Style from display definition.
+    func getPreferredLocalizedIssuerDisplayDefinition() -> RequesterStyle
+    {
+        guard let definitions = display else
```

Comment:
Does this perform the null check for display property?

Reply:
yep!